vector/src/main/res/values/strings.xml

```diff
@@ -2526,18 +2526,26 @@
     <string name="ftue_auth_carousel_2_title">You\'re in control.</string>
     <!-- TODO TO BE REMOVED -->
     <string name="ftue_auth_carousel_2_body">Element lets you choose where your messages are stored, keeping you in control of your data.</string>
-    <string name="ftue_auth_carousel_3_title">Secure Messaging.</string>
+    <string name="ftue_auth_carousel_3_title">Secure messaging.</string>
```

Comment:
I see the change from `M` to `m`. 👍 

FTR On the stores we use "Secure Messenger". [Source](https://github.com/vector-im/element-android/blob/main/fastlane/metadata/android/en-US/title.txt#L1)

<img width="738" alt="image" src="https://user-images.githubusercontent.com/3940906/150117513-048924a8-a036-4241-ae5f-8cf390d37dd9.png">

<img width="660" alt="image" src="https://user-images.githubusercontent.com/3940906/150117554-44a6ebe2-56ae-49b5-a461-d7e1b6b1e10c.png">

Could be nice to harmonize? CC @amshakal @daniellekirkwood